example/cdn/index.html

```diff
@@ -0,0 +1,267 @@
+                            placeholder="0C55:00D5:0922:0234" oninput="this.value = this.placeholder" />
+                    </fieldset>
+                </div>
+                <div class="grid">
+                    <fieldset>
+                    </fieldset>
+                    <fieldset>
+                        <label for="voxel">Voxel coordinates</label>
+                        <input id="portaltoVoxel" type="text" name="voxel" placeholder="tbd"
+                            oninput="this.value = this.placeholder" />
+                    </fieldset>
+                </div>
+            </form>
+
+            <br />
+            <form>
+                <div class="grid">
+                    <fieldset>
+                        <label for="galactic">Galactic coordinates</label>
+                        <input id="galactic" type="text" name="galactic" placeholder="0A32:00D5:0F44:0234"
+                            oninput="onGalacticCoord()" value="0A32:00D5:0F44:0234" />
+                        <small class="help-text"><i>Valid characters are 0123456789ABCDEF with semicolons.</i></small>
+                    </fieldset>
+                    <fieldset>
+                        <label for="portal">Portal coordinates</label>
+                        <input id="galactictoGlyph" type="text" name="portal" placeholder="023456123456"
+                            oninput="this.value = this.placeholder" />
+                    </fieldset>
+                </div>
+                <div class="grid">
+                    <fieldset>
+                    </fieldset>
+                    <fieldset>
+                        <label for="voxel">Voxel coordinates</label>
+                        <input id="galactictoVoxel" type="text" name="voxel" placeholder="tbd"
+                            oninput="this.value = this.placeholder" />
+                    </fieldset>
+                </div>
+            </form>
+
+            <br />
+            <form>
+                <div class="grid">
+                    <fieldset>
+                        <label for="voxel">Voxel coordinates</label>
+                        <input id="voxel" type="text" name="voxel" oninput="onVoxelCoord()" placeholder="tbd" />
+                        <small class="help-text"><i>This input takes in an object representing a voxel
+                                coordinate.</i></small>
+                    </fieldset>
+                    <fieldset>
+                        <label for="portal">Portal coordinates</label>
+                        <input id="voxeltoGlyph" type="text" name="portal" placeholder="023456123456"
+                            oninput="this.value = this.placeholder" />
+                    </fieldset>
+                </div>
+                <div class="grid">
+                    <fieldset>
+                    </fieldset>
+                    <fieldset>
+                        <label for="galactic">Galactic coordinates</label>
```

Comment:
These output input labels shouldn't have the `for` attribute.
And since these outputs cannot be edited, they should be made readonly (or shouldn't use input fields in the first place)

Reply:
Removed the input elements being used for output 😋